SimplyPluralApiBodies.yml

```diff
@@ -474,21 +412,63 @@ components:
         color:
           description:
             $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/color/description"
-          type: string
-        private:
+          type:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/color/type"
+        supportDescMarkdown:
           description:
-            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/private/description"
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/supportDescMarkdown/description"
+          type:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/supportDescMarkdown/type"
+
+    customFrontContainer:
+      type: object
+      properties:
+        exists:
           type: boolean
-        preventTrusted:
+        id:
+          description: The custom front Id
+          type: string
+        content:
+          description: The custom front content
+          $ref: "#/components/schemas/customFrontInfo"
+
+    customFrontInfo:
+      type: object
+      properties:
+        $ref: "#/components/schemas/customFront/properties"
+        buckets:
+          description: The privacy buckets for this custom front
+          type: array
+          items:
+            type: string
+        avatarUuid:
+          description: Id of the uploaded avatar
+          type: string
+        frame:
+          $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/frame"
+        uid:
           description:
-            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/preventTrusted/description"
-          type: boolean
-        supportDescMarkdown:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/uid/description"
+          type:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/uid/type"
+        lastOperationTime:
           description:
-            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/supportDescMarkdown/description"
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/lastOperationTime/description"
+          type:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/lastOperationTime/type"
+
+    customFrontSend:
+      type: object
+      properties:
+        $ref: "#/components/schemas/customFront/properties"
+        private:
+          description: Whether this custom front is private
```

Comment:
Whether this custom front is private (Legacy: Version 1.9 and lower)

Seems better, as this property is not relevant in 1.11 and up.